package.json

```diff
@@ -14,35 +14,35 @@
   },
   "main": "index.js",
   "scripts": {
-    "start": "NODE_ENV=production node --trace-warnings ./src/server/server.babel.js",
+    "start": "NODE_ENV=${NODE_ENV:-production} node --trace-warnings ./src/server/server.babel.js",
```

Comment:
what's this do?

Reply:
If there's a `NODE_ENV` already bound in the environment, it uses that value, otherwise `production`.

Reply:
can you give me an example where this is useful? it kinda scares me because this makes it seem a little less expressive because after calling this, i still don't know what env it's running in.

Reply:
I could revert this change if you like - it's a holdover from when I was associating the value of `NODE_ENV` with the `.env.<NODE_ENV>` files.

That said, here's an example of how it'd work:

```sh
$ # when NODE_ENV='' (unbound)
$ npm start # NODE_ENV is bound to 'production'
$ # when NODE_ENV='production'
$ npm start # NODE_ENV is bound to 'production'
$ # when NODE_ENV='development'
$ npm start # NODE_ENV is bound to 'development'
$ # ...etc
```